app/mockServices/mockAPI.service.ts

```diff
@@ -0,0 +1,42 @@
+interface IAPIService {
+  kindToResource(kind: string, humanize: boolean): string;
+}
+
+/** Backend service communications. */
+export class APIService implements IAPIService {
+  public static $inject = ['$filter'];
+
+  private $filter : any;
+
+  constructor ($filter: any) {
+    this.$filter = $filter;
+  }
+
+  public kindToResource(kind: string, humanize: boolean) {
```

Comment:
When we are in mock mode, we do not include the common services at all. Doing so requires a bunch of checks that fail when not having a backend to connect to. So, we do need to mock any particular service calls that get used even when we are in mock mode.